src/components/LogIn.js

```diff
@@ -4,12 +4,13 @@ import { Redirect } from 'react-router-dom';
 import * as firebase from 'firebase';
 import { FirebaseAuth } from 'react-firebaseui';
 
+import Loading from './Loading';
 import { auth } from '../firebaseInit';
-import { getAuthMemberDoc } from '../connectors';
+import { getAuthMemberDocIsLoaded, getAuthMemberDoc } from '../connectors';
 
-const LogIn = ({ authMemberDoc, noRedirect }) => {
+const LogIn = ({ authFirebaseUser, authMemberDocIsLoaded, authMemberDoc, noRedirect }) => {
   const uiConfig = {
-    signInFlow: 'popup',
+    signInFlow: 'popup', // TODO will 'redirect' fix some of our issues?
```

Comment:
Thought in general on changing this to `redirect`? Personally I prefer not having the additional windows

Reply:
Sounds fine to me.